src/shared/install.c

```diff
@@ -1873,6 +1874,12 @@ static int install_info_symlink_wants(
                 if (q < 0)
                         return q;
 
+                q = unit_file_exists(scope, paths, dst);
```

Comment:
This should be below the validity check below.

`continue` is the wrong thing to do. It is totally valid to create symlinks before the target exists, even if most of the time it is a user mistake.

Also, this is run both server-side and user-side, and I don't think writing with log_warning is appropriate in the first case.

Reply:
I would like to be able to show a warning to the user when he runs `systemctl enable service`. What's the preferred way to do that in this particular case?